### PR TITLE
feat(data-qa-id): add data-qa-id props for some components #patch

### PR DESCRIPTION
**Pull request by kraiztmatte**
adding some data-qa-id props to support front-end test automation

### CODE REVIEW

File: src/components/Box/Box.tsx [hunk 1]
```diff
@@ -10,11 +10,12 @@ import "./index.scss";
 export interface BoxProps {
   children: React.ReactNode;
   title: string;
+  dataQA?: string | null;
```
Comment (aris-fm) [inline src/components/Box/Box.tsx:13]: Please rename `dataQA` prop to `dataQa` to be consistent with prop in [other components](https://github.com/AccelByte/justice-ui-library/blob/master/src/components/Badge/Badge.tsx#L21)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [0.13.5]
```
Comment (aylwin-AB) [inline CHANGELOG.md]: I think you need to rebase this branch, and up the version

also please update version at `package.json` also

**Comment by rayhan-ab** [comment]
please make sure to run yarn build and commit the changes in /build

File: src/components/Box/Box.tsx (hunk 1 above)
Comment (aylwin-AB) [inline src/components/Box/Box.tsx:13]: I think we need change the props name to `dataQa` to keep it consistance with other components

Comment (kraiztmatte) [reply src/components/Box/Box.tsx]: changed, thankyou

File: CHANGELOG.md (hunk 2 above)
Comment (kraiztmatte) [reply CHANGELOG.md]: rebased, and update the package.json

File: src/components/Box/Box.tsx (hunk 1 above)
Comment (kraiztmatte) [reply src/components/Box/Box.tsx]: changed, thankyou

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [0.13.6]
+### Added
+- add data-qa-id props for `Box`, `ErrorWrapper`, `InlinePopover`, `LoadingIndicator`, `loadOrErrorWrapper`, `PureSeachPanel`, `Spinner`, `Tag`, `Tooltip` component
```
Comment (rayhan-ab) [inline CHANGELOG.md:10]: `LoadingOrErrorWrapper`?

Comment (kraiztmatte) [reply CHANGELOG.md]: updated, thankyou

**Review by rayhan-ab — APPROVED** [review]

File: src/components/Box/Box.tsx [hunk 4]
```diff
@@ -10,11 +10,12 @@ import "./index.scss";
 export interface BoxProps {
   children: React.ReactNode;
   title: string;
+  dataQa?: string | null;
```
Comment (castrix) [inline src/components/Box/Box.tsx:13]: does it really need `null` type here? I noticed that it already has optional `?` type

Comment (rayhan-ab) [reply src/components/Box/Box.tsx:13]: yes. `qa` util in AP returns string | null

**Review by castrix — APPROVED** [review]

**Pull request merged**